contracts/PvpArena.sol

```diff
@@ -370,7 +373,7 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
     }
 
     /// @dev performs all queued duels
-    function performDuels(uint256[] calldata attackerIDs) external restricted {
+    function performDuels(uint256[] memory attackerIDs) public restricted {
```

Comment:
this probably makes no difference but it's worth knowing that in some cases, external functions can handle arrays of data more efficiently (in terms of gas). An extra internal function might be ideal here considering that the restricted modifier is going to run (adding to gas) when called from the restartRankedSeason(), which is already checked against restriction. Not big enough of a deal to bother with this know, but it's worth mentioning.

Reply:
Noted!